tools/c7n_azure/c7n_azure/resources/cdn.py

```diff
@@ -40,3 +42,32 @@ class resource_type(ArmResourceManager.resource_type):
             'sku.name'
         )
         resource_type = 'Microsoft.Cdn/profiles'
+
+@CdnProfile.filter_registry.register('waf')
+class WebAppFirewallFilter(Filter):
+    """Check waf enabled/disabled on cdn profiles
+
+    :example:
+
+    .. code-block:: yaml
+
+        policies:
+            name: test-waf-not-enabled
```

Comment:
policies is an array, so this is failing the meta test that validates doc examples.